go.mod

```diff
@@ -284,4 +284,4 @@ replace (
 	k8s.io/klog/v2 => github.com/simonpasquier/klog-gokit/v3 v3.0.0
 )
 
-go 1.18
+go 1.19
```

Comment:
Is this necessary? Do we depend on any 1.19 features?

Reply:
No, it's not necessary.